.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="GoogleStyle" />
```

Comment:
Hey, is this suppose to be in this commit? If yes, why is it necessary? If no, please remove it

- [ ] Explain or remove it

Reply:
@leinardi 
Hi, unfortunately this is added by my personal Android Studio configuration. And every time I try to remove it, it gets added automatically. Anyways, it won't really affect any build or performance. Would you mind removing it yourself after the PR is merged?